examples/com.eclipsesource.modelserver.example/src/main/java/com/eclipsesource/modelserver/example/ExampleServerLauncher.java

```diff
@@ -31,7 +31,7 @@
 	private static String WORKSPACE_ROOT = "workspace";
 	private static String ECORE_TEST_FILE = "Coffe.ecore";
 	private static String COFFEE_TEST_FILE = "SuperBrewer3000";
-	private static String COFFEE_XMI_TEST_FILE = COFFEE_TEST_FILE + ".xmi";
+	private static String COFFEE_COFFEE_TEST_FILE = COFFEE_TEST_FILE + ".coffee";
```

Comment:
That name is a bit weird :) Maybe we can rename `COFFEE_TEST_FILE` to something else in order to have this fixed?